InGreedIoApi/Migrations/20240606200852_ResolveMigrations.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace InGreedIoApi.Migrations
+{
+    /// <inheritdoc />
+    public partial class ResolveMigrations : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+
+        }
+
+        /// <inheritdoc />
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+
+        }
+    }
+}
```

Comment:
What is the purpose of the empty migration?

Reply:
first migration